web/server/api/endpoints/models.py

```diff
@@ -24,14 +23,6 @@ def get_models(
     context: Context = Depends(get_loaded_context),
 ) -> t.List[models.Model]:
     """Get a mapping of model names to model metadata"""
-    try:
-        context.load()
-    except Exception:
-        raise ApiException(
-            message="Unable to get models",
-            origin="API -> models -> get_models",
-        )
-
```

Comment:
Why is this removed? Is it because the file watcher reloads the context on file changes now?

Reply:
yes, file watcher reloads the context on  `model`  or `seed` file changes and collecting models to send over SSE